xbmc/osx/XBMCHelper.cpp

```diff
@@ -114,7 +114,7 @@ bool XBMCHelper::OnSettingChanging(const CSetting *setting)
     if (IsRunning() && GetMode() != remoteMode)
     {
       bool cancelled;
-      if (!CGUIDialogYesNo::ShowAndGetInput(13144, 13145, 13146, 13147, -1, -1, cancelled, 10000))
+      if (!CGUIDialogYesNo::ShowAndGetInput(13144, 13145, cancelled, -1, -1, 10000))
```

Comment:
sure, this needs some cleaup/changes to the dialog, but that's stuff for another PR I guess